README.md

```diff
@@ -12,47 +12,40 @@
   - [Prerequisites](#prerequisites)
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Ok.  WIll do that.  Thanks